packages/manager/modules/nasha/src/components/partition/create/create.controller.js

```diff
@@ -46,4 +51,9 @@ export default class NashaComponentsPartitionCreateController {
       )
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnClose() {
```

Comment:
Same as https://github.com/ovh/manager/pull/7762/files#r943141172

Reply:
It's applied.